### PR TITLE
Fix login screen, fix null priority

### PR DESCRIPTION
**Pull request by joeyyang**
This PR addresses several issues:
- [Login issue](https://github.com/firebase/vulcan/issues/32): This PR includes the Firebase client through Bower rather than through npm and includes `https://*.firebaseio.com` as a valid domain.
- Null priority issue: Adding new data was failing if the priority field was left blank. This PR correctly handles the null case.
- bumps semver: from `1.3.1` to `1.3.2`

### CODE REVIEW

**Comment by joeyyang** [comment]
@tstirrat — assigning to you! Let me know if you have any questions.

File: bower.json [hunk 1]
```diff
@@ -16,5 +16,9 @@
     "bower_components",
     "test",
     "tests"
-  ]
+  ],
+  "dependencies": {
+    "react": "0.8.0",
```
Comment (tstirrat) [inline bower.json]: is react needed, we are still including it via browserify aren't we?

**Comment by joeyyang** [comment]
I signed it?

**Comment by joeyyang** [comment]
@tstirrat back to you!

**Pull request merged**